config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,39 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  labels:
+    app.kuberentes.io/name: issuer
```

Comment:
please add a comment here that it is spelled correctly, and point to the codespell config

Reply:
this is auto generated file adding commit might get overriden when its rebuild